packages/wpcom.js/package.json

```diff
@@ -44,6 +44,7 @@
 		"qs": "^6.5.2"
 	},
 	"devDependencies": {
+		"wpcom-oauth-cors": "^1.0.2-beta",
```

Comment:
Confirming it makes sense to be a dev dependency as it's practically used only in tests.